README.md

```diff
@@ -8,7 +8,7 @@ A significant portion of the Security Working Group's mission is achieved by mai
 * _TBD:  list of project names_
 
 ### Adding subprojects
-To request that the Security Working Group take on ownership and maintainership of a particular project, create a new issue in this repository using the appropriate issue template. You may be requested to present your proposal at the next Security Working Group meeting. Approvers will vote on whether or not the Working Group will accept the project.
+To request that the Security Working Group take on ownership and maintainership of a particular project, create a new issue in this repository using the appropriate issue template. You may be requested to present your proposal at the next Security Working Group meeting. The Working Group will accept the project upon unanimous agreement by Approvers.
```

Comment:
Off topic nitpick: please follow the ROS 2 guidelines: each sentence should start on a new line (to avoid this kind of omnibus diffs).

Reply:
I'll make a follow-up PR.